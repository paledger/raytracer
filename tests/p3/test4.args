pixeltrace ../resources/refract2.pov 640 480 370 270
