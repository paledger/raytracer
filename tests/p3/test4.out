Pixel: [370, 270] Color: (60, 18, 61)
o - Iteration type: Primary
|   Ray: {0 0 14} -> {0.1044 0.06307 -0.9925}
|   Hit Object ID (1 - Sphere) at T = 11.43, Intersection = {1.193 0.7208 2.656}
|   Normal {0.3978 0.2403 0.8854}
|   Transformed Ray: {0 0 14} -> {0.1044 0.06307 -0.9925}
|   Ambient: 0.1, 0, 0.1
|   Diffuse: 0.06088, 0, 0.06088
|   Specular: 0, 0, 0
|
|\
| o - Iteration type: Refraction
| |   Ray: {1.193 0.7208 2.655} -> {-0.03511 -0.0212 -0.9992}
| |   Hit Object ID (2 - Sphere) at T = 4.607, Intersection = {1.032 0.6231 -1.948}
| |   Normal {-0.421 -0.1639 0.8921}
| |   Transformed Ray: {1.193 0.7208 2.655} -> {-0.03511 -0.0212 -0.9992}
| |   Ambient: 0.152, 0.276, 0.16
| |   Diffuse: 0, 0, 0
| |   Specular: 0, 0, 0
| |   Extra Info: into-object 
| |----
| 
 \
  o - Iteration type: Reflection
  |   Ray: {1.193 0.7208 2.656} -> {0.7585 0.4581 0.4634}
  |   No intersection.
  |----
  

--------------------------------------------------------------------------------

